examples/Cargo.toml

```diff
@@ -0,0 +1,163 @@
+[package]
+name = "gfx-examples"
+version = "0.7.0"
```

Comment:
should be 0.1.0
also, quite a few of the fields are not needed here since this project is never going to be published